src/runtime/contrib/tensorrt/tensorrt_module.cc

```diff
@@ -228,6 +253,17 @@ class TensorRTModule : public runtime::ModuleNode {
     } else {
       CHECK(context->executeV2(bindings.data())) << "Running TensorRT failed.";
     }
+    for (size_t i = 0; i < num_outputs; ++i) {
+      const int index_in_inputs = inputs.size() - num_outputs + i;
+      DLTensor* out_arg = inputs[index_in_inputs];
+      int binding_index = engine->getBindingIndex(engine_and_context->outputs[i].c_str());
+      CHECK_NE(binding_index, -1);
+      if (out_arg->ctx.device_type == kDLCPU) {
```

Comment:
Make this `out_arg->ctx.device_type != kDLGPU`

Reply:
Done.